Exec/science/flame_wave/inputs_He/inputs.He.25cm.dynamic.1000Hz

```diff
@@ -77,26 +104,32 @@ amr.v                 = 1       # verbosity in Amr.cpp
 # REFINEMENT / REGRIDDING
 amr.max_level       = 2       # maximum level number allowed
 amr.ref_ratio       = 4 2 2 2 # refinement ratio
-amr.regrid_int      = 2 2 2 2 # how often to regrid
+amr.regrid_int      = 0       # static grids ftw
```

Comment:
This one is named dynamic, is this really what you want or is this a mistake?

Reply:
fixed